addons/mail/tests/test_res_users.py

```diff
@@ -75,20 +74,19 @@ def test_notification_type_convert_internal_inbox_to_portal(self):
 
         # Change the internal user to portal, and make sure it automatically converts from inbox to email notifications
         user.write({'groups_id': [
-            Command.unlink(self.env.ref('base.group_user').id),
-            Command.link(self.env.ref('base.group_portal').id),
+            (3, self.env.ref('base.group_user').id),
+            (4, self.env.ref('base.group_portal').id),
```

Comment:
Hello guys, just curious about why using directly the number (3 and 4) instead of the `Command` is considered an improvement on this commit?

Reply:
It is not ... I had conflicts, removed Command when resolving the conflict, then had an issue with Command usage and fall back on using numbers as I find it easier. But that should effectively not be done in stable, even if not really annoying in tests.